.github/workflows/integration.yaml

```diff
@@ -167,3 +167,64 @@ jobs:
             unicode \
             unicode/utf16 \
             unicode/utf8
+
+
+  wasi-testsuite:
+    name: wasi-testsuite
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-12, windows-2022]
+
+    steps:
+      - uses: actions/cache@v3
+        id: cache
+        with:
+          path:
+            ~/.cache/go-build
+          key: integration-test-wasi-testsuite-${{ env.GO_VERSION }}
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Checkout wazero
+        uses: actions/checkout@v3
+
+      - name: Install wazero
+        run: |
+          go install ./cmd/wazero
+        shell: bash
+
+      - name: Checkout wasi-testsuite
+        uses: actions/checkout@v3
+        with:
+          repository: evacchi/wasi-testsuite
+          ref: gha/wazero-win
+      
+      - name: Initialize Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
comment if this can be any version of python (e.g. `. # latest version of python 3` if it isn't specific)